mifospay/src/main/res/layout/activity_edit_profile.xml

```diff
@@ -49,31 +49,51 @@
                     android:layout_marginTop="@dimen/marginItemsInSectionLarge"
                     android:layout_width="match_parent"
-                    app:helperText="@string/username"
-                    android:hint="@string/username"
                     android:layout_height="wrap_content" >
 
                     <android.support.design.widget.TextInputEditText
                         android:id="@+id/et_edit_profile_username"
                         android:layout_width="match_parent"
-                        android:layout_height="wrap_content" />
+                        android:layout_height="wrap_content"
+                        android:drawableLeft="@drawable/ic_person_black_24dp"
+                        android:drawableStart="@drawable/ic_person_black_24dp"
+                        android:drawablePadding="10dp"
+                        android:drawableTint="@color/colorAccent"
+                        android:hint="@string/username"
+                        android:inputType="textPersonName|text"
+                        android:paddingStart="10dp"
+                        android:paddingRight="0dp"
+                        android:paddingEnd="0dp"
+                        android:paddingLeft="10dp"
+                        android:textSize="14sp"/>
 
                 </android.support.design.widget.TextInputLayout>
 
+
+
                 <android.support.design.widget.TextInputLayout
                     style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
                     android:layout_marginTop="@dimen/marginItemsInSectionMedium"
                     android:layout_width="match_parent"
                     android:id="@+id/til_edit_profile_email"
                     app:layout_anchor="@id/til_edit_profile_username"
-                    app:helperText="@string/email"
-                    android:hint="@string/email"
                     android:layout_height="wrap_content" >
 
                     <android.support.design.widget.TextInputEditText
                         android:id="@+id/et_edit_profile_email"
                         android:layout_width="match_parent"
-                        android:layout_height="wrap_content" />
+                        android:layout_height="wrap_content"
+                        android:drawableLeft="@drawable/ic_email_black_24dp"
+                        android:drawableStart="@drawable/ic_email_black_24dp"
+                        android:drawablePadding="10dp"
+                        android:drawableTint="@color/colorAccent"
+                        android:hint="@string/email"
+                        android:inputType="textEmailAddress"
+                        android:paddingLeft="10dp"
+                        android:paddingRight="0dp"
+                        android:paddingEnd="0dp"
+                        android:paddingStart="10dp"
+                        android:textSize="14sp"/>
```

Comment:
@tacitonic don't use hardcoded values it's not considered as a good practice. Instead, you can define your dimension inside dimens folder an use it as @dimens/value_10dp something as per you name them.